packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Shape/Shape.tsx

```diff
@@ -28,25 +27,27 @@ const shapeOptions = makeItems([
   'shape-line'
 ])
 
-interface ShapeProps extends Omit<ToolbarGroupItemProps, 'id' | 'options'> {}
+interface ShapeProps extends Omit<ToolbarGroupItemProps, 'id' | 'options'> {
+  height?: number
+}
 interface ShapeCallProps extends ToolbarGroupItemCallProps {}
 
 type Props = ShapeProps & ShapeCallProps
 
 const Shape = (props: Props) => {
-  const collapseRGroup = useMediaQuery({ query: '(max-height: 850px)' })
+  const { height, ...rest } = props
 
-  if (collapseRGroup) {
+  if (height && height <= 850) {
```

Comment:
Please move to  constants

Reply:
fixed in [move media sizes to constants](https://github.com/epam/ketcher/pull/323/commits/d9abe72d289ea5e2293030fbaa0060c266759e0c)